Assets/Scripts/Edges/EdgeGameObjectFactory.cs

```diff
@@ -50,5 +48,22 @@ void CreateNewEdgeGameObjectFor(Edge edge, IList<Vector3> edgePoints)
       lineRenderer.positionCount = positionCount;
       return (line, lineRenderer);
     }
+
+    private void GenerateCollider(GameObject line, Vector3 firstEdgePoint, Vector3 secondEdgePoint)
+    {
+      var collider = new GameObject(Constants.ColliderGameObjectName);
+      collider.transform.parent = line.transform;
+      var capsule = collider.AddComponent<CapsuleCollider>();
+      collider.transform.parent = capsule.transform;
+
+      capsule.radius = 0.09f;
+      capsule.height = Vector3.Distance(secondEdgePoint, firstEdgePoint) + Constants.AdditionalColliderLength;
+      capsule.direction = 2;
```

Comment:
comment like this would be helpful:
"The value can be 0, 1 or 2 corresponding to the X, Y and Z axes, respectively."